### PR TITLE
TAJO-1659: Simplify scan iteration in SeqScan.

### PR DESCRIPTION
**Pull request by hyunsik**
[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/612)

### CODE REVIEW

File: tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java [hunk 1]
```diff
@@ -228,26 +237,15 @@ public Tuple next() throws IOException {
...
-    } else {
-      while ((tuple = scanner.next()) != null) {
-        if (qual.eval(tuple).isTrue()) {
-          projector.eval(tuple, outTuple);
-          return outTuple;
-        }
-      }
-      return null;
+    while(scanIt.hasNext()) {
+      Tuple outTuple = new VTuple(outColumnNum);
```
Comment (jihoonson) [inline tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java]: outTuple don't have to be created for every input row.

File: tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/HashShuffleAppenderManager.java [hunk 2]
```diff
@@ -30,9 +30,11 @@
 import org.apache.tajo.catalog.TableMeta;
 import org.apache.tajo.conf.TajoConf;
 import org.apache.tajo.conf.TajoConf.ConfVars;
+import org.apache.tajo.util.Pair;
```
Comment (jihoonson) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/HashShuffleAppenderManager.java]: Please remove unused imports.

**Comment by jihoonson** [comment]
+1. I left trivial comments. Please consider them before commit.

File: tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java (hunk 1 above)
Comment (hyunsik) [inline tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java]: It is actually for every row. While condition is just for check if there is next row, and every while loop will be just returned for each tuple.

**Comment by hyunsik** [comment]
After merging another patch, it is causing unit test failures. I'll check them and then I'll commit.

**Pull request closed without merging**